submissions/Olumide Ologbon/html/signup.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            signup
+        </title>  
+        <link href = "style.css" rel = "stylesheet">
+    </head>
+    <header class="head">
+        <div>
+            <div class="dropdown">
+                <button class="dropbtn">More</button>
+                <div class="dropdown-ctn">
+                    <a href="index.html">Home</a>
+                    <a href="#">About</a>
+                    <a href="#">Clients</a>
+                    <a href="#">Contact Us</a>
+                </div>
+            </div>   
+            <div class="dropdown">
+                <h1 style="margin-left: 550px; color: black;"> 
+                    Signup Page
+                </h1>                   
+            </div>
+            <div class="dropdown">
+                <button class="dropbtn" style="margin-left: 450px ;">Account</button>
+                <div class="dropdown-ctn" style="margin-left: 450px ;">
+                        <a href="signup.html">Sign up </a>
+                        <a href="#">Login</a>
+                        <a href="#">Track Orders</a>
+                        <a href="#">Wish List</a>
+                </div>
+                    </div>    
+                                
+                    
+        </div>        
+    </header>        
+    <body>    
+        <div class="signupbox">
+            <form> 
+                <h3 class="create">
+                    Create an Account
+                </h3>
+                Fullname:<br> 
+                <input type="text" name="fullname" placeholder="Surname First"><br> 
+                Username:<br> 
+                <input type="text" name="username" placeholder="Set username"><br>                  
+                Email:<br> 
+                <input type="text" name="email" placeholder="Enter Email address"> <br> 
+                Password:<br> 
+                <input type="password" name="password" placeholder="Enter password">
+                <br><br> 
+                <input type="submit" value="Create an Account"> 
+                <h4>
+                    Already have an account?<a href="login.html">Log in</a>
+                </h4>
+            </form>
+        </div>  
+        </body>
+    <footer class="foot">
+        <p>
+            (c) Copyright 2019 Shopal
```

Comment:
You can code the copyright sign instead of hard coding it.
Check out this link for help https://www.w3schools.com/html/html_symbols.asp